src/components/MerchantPayout/VoucherStatusModal/LoadingCard.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from "react";
+import { StyleSheet } from "react-native";
+import { Card } from "../../Layout/Card";
+import { color } from "../../../common/styles";
+import { LoadingView } from "../../Loading";
+
+const styles = StyleSheet.create({
+  card: {
+    width: 512,
+    height: 512,
+    maxWidth: "100%",
+    maxHeight: "50%",
+    backgroundColor: color("grey", 0)
+  }
+});
+
+export const LoadingCard: FunctionComponent = () => {
+  return (
+    <Card style={[styles.card]}>
+      <LoadingView />
+    </Card>
+  );
+};
```

Comment:
Forgot to mention, the loading screen will just be an ActivityIndicator, so no need a card

Reply:
So a greyed out background with no card? or just replace the LoadingView with the ActivityIndicator?

Reply:
Yes greyed out background, no card, white ActivityIndicator in the middle. You can also update LoadingView to include a color prop.